app.js

```diff
@@ -11,6 +12,8 @@ app.get("/api", (request, response) => {
   response.status(200).send(endpoints);
 });
 
+app.get("/api/articles/:article_id", getArticleById);
+
 app.use((err, request, response, next) => {
   const unhandledError = err;
   console.log({ unhandledError });
```

Comment:
Looks like you started some error handling here. Please do add the tests and add your error handlers to get them passing 😁